swagger/admin.yaml

```
@@ -460,6 +499,23 @@ definitions:
       active:
         type: boolean
         x-nullable: true
+      roles:
+        type: array
+        items:
+          $ref: '#/definitions/OfficeUserRolePayload'
+  OfficeUserRolePayload:
```

Let's remove the `Payload` suffix from this definition:

```suggestion
  OfficeUserRole:
```

---

done